articles/typography/justification.en.html

```diff
@@ -264,7 +264,7 @@ <h3>Other Writing Systems</h3>
         <p class="note">Readers who wish to provide such “more specific information”
           are invited (and strongly encouraged) to 
           <a class="interaction" target="_blank" href="https://github.com/w3c/i18n-drafts/issues/new?title=%5Barticles/typography/justification%5D%20specific_title_here&amp;body=%5Bsource%5D%20(http%3A%2F%2Flocalhost%2Fgit%2Fi18n-drafts%2Farticles%2Ftypography%2Fjustification.en.html)%20%5Ben%5D%0A%0Ayour_comment_here">create a github issue</a>
```

Comment:
Our preview is broken so I can't tell... but `http` and `localhost` (for the URL in the parentheses) might be a mistake?